.github/workflows/gradle.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           path: ${{env.BUILD_CACHE_PATH}}
           key: ${{github.run_id}}
       - name: Compile & Static Analysis
```

Comment:
Since we are moving static analysis to `unit_other`, this should be changed to just `Compile`.